wiki/osu!tourney/kr.md

```diff
@@ -0,0 +1,353 @@
+-   `background-win2` (optional) - 배경색은 어둡게 됩니다. 오른쪽 팀이 이겼을 때 결과화면에서 이 사진으로 사라집니다. 결과 화면을 떠날 때 백그라운드로 다시 표시됩니다. 이 사진이 배치되지 않을 경우 '배경'이 그대로 유지됩니다.
+-   `tourney-title` (optional) - 이 영상은 배경 화면 맨 위와 아래쪽에 위치합니다. 이는 로고의 로고를 고정하는 데 사용할 수 있습니다(이것은 영상으로만 작동합니다.)
+
+이것에 최적화된 해상도는 1280x720 60FPS입니다.
+
+![osu!tourney with team icons. The left icon is named Team 1.png and the right icon is named Team 2.png](Osutourneyicons.png "fig:osu!tourney with team icons. The left icon is named Team 1.png and the right icon is named Team 2.png")
+
+(예: 국기 또는 사용자 아바타)을 skin폴더에 생성할 수 있습니다. 아이콘은 반드시 /osu!/Skins/User/tournament/icons  `.../osu!/Skins/User/tournament/icons`. 이 경로로 배치해야 합니다. 또한 모든 아이콘은 팀 이름과 정확히 일치해야 합니다.
+
+만약 당신의 경기가 Test Tourney: '(Team 1) vs (Team 2)' 라 불린다면 당신은 그 각각의 팀의 아이콘들은 반드시'(team 1)과 (team 2)' 불리어야 합니다. 또한 아이콘은 `*.jpg`또는 `*.gif`형식으로 되어 있으며, '50x50 최적 해상도를 가지고 있어야 합니다.
+
+------------------------------------------------------------------------
+
+문제 해결 
+===============
+
+**a) 새로운 Osu!는 어떻게 만드나요? 현제 설치된 Osu!를 제거하지 않고 어떻게 설치합니까?**
+
+- Osu!.exe를 빈 폴더에 복사하여 붙여넣기를 하세요. 이는 많은 문제를 해결합니다! osu!tourney를 별도로 실행하는 것이 좋습니다!
+
+<!-- -->
+
+**b) 나의 Osu! 인스턴스가 제대로 정렬되지 않습니다.**
+
+osu! tourney는 기본 모니터에서만 실행됩니다. 보조모니터를 비 활성화하십시오! 또한 화면 해상도가 너무 낮을 수 있습니다.
+
+관리자모드로 osu를 실행하지 않도록하십시오! (osu!가 직접 요구하는 경우는 제외). 최신 빌드로 업데이트가 되었는지 확인 해주세요.
+
+<!-- -->
+
+**c) 나에게 오류 메시지를 던지고 osu!tourney가 열리지 않습니다. / 또는 다시 종료됩니다!**
+
+-   [Tournament Staff](https://osu.ppy.sh/g/26) 용자 그룹에 속하는지 확인하십시오. 사용자 그룹. 대부분의 경우에는 일시적으로 추가되므로 액세스가 만료되었을 수 있습니다.
+-   tournament.cfg의 privateserver 키에 값이 설정되어 있지 않은지 확인해야합니다.
+-   osu에 로그인했는지 확인하세요. 그렇지 않으면 osu를 실행한 후, 자동로그인을 한 다음에 osu를 닫은 다음 다시 토너먼트 클라이언트로 다시 시작하세요.
+-   하나의 모니터만 사용하세요.
+-   관리자모드로 osu를 실행하지 않도록하십시오! (osu!가 직접 요구하는 경우는 제외). 
+-   최신 빌드로 업데이트가 되었는지 확인 해주세요.
+
+<!-- -->
+
+**d) Osu! 노래 데이터베이스가 손상됩니다!**
+
+새로운 Osu!를 설치하고, 사용하세요.
+
+<!-- -->
+
+**e) osu!tourney의 배경이 보이지 않습니다!**
+
+배경파일이 다음위치에 있는지 확인하십시오. `.../osu!/Skins/User/tournament/background.png`. See **[스킨](Osu!tourney#Skinning "wikilink")**.
+
+<!-- -->
+
+**f) 어떻게 osu를 어떻게 다시 노말 클라이언트로 바꿀 수 있습니까?**
+
+`tournament.cfg` 파일의 이름을 변경하거나 삭제하십시오.
+
+<!-- -->
+
+**g) 나의 클라이언트가 관전이 안 된다!**
+
+Osu!에 들어 왔는지 확인하십시오! 굵은 글씨가 될 때까지 검정색 패널의 방 이름을 클릭하십시오. 클라이언트가 여전히 관중이 아닌 경우 '패닉(panic)' 버튼을 클릭하십시오.
+
+<!-- -->
+
+**h) 방들이 나타나지 않습니다!**
+
+-   멀티방의 이름이 잘못되었거나 'tournament.cfg' 잘못된 약어가 사용되었습니다. 
+-   정확한 멀티방 이름을 사용했는지 확인하십시오. (see **[매칭 설정](Osu!tourney#Match_creation "wikilink")**). 참조하십시오.
+-   실수한 경우 방을 다시 만듭니다.
+
+<!-- -->
+
+**i) 팀 이름이 보이지 않습니다!**
+
+-   멀티방의 이름이 잘못되었거나 'tournament.cfg' 잘못된 약어가 사용되었습니다. 
+-   정확한 멀티방 이름을 사용했는지 확인하십시오. (see **[매칭 설정](Osu!tourney#Match_creation "wikilink")**). 참조하십시오.
+-   실수한 경우 방을 다시 만듭니다.
+
+<!-- -->
+
+**j) Twitch.tv와 같은 서비스에 내 토너먼트를 스트리밍하려면 어떻게 해야 합니까?**
+
+스트리밍을 설정하려면 Open Broadcaster Software를 사용하는 것이 좋지만 액세스 권한이 있는 다른 스트리밍 클라이언트를 자유롭게 사용할 수도 있습니다. osu! tourney의 윗부분의 해상도는 1280x720입니다. 검은 색 컨트롤 패널을 스트리밍하지 마십시오.
+
+<!-- -->
```

Comment:
Replace All `<!-- -->` with empty space. One of the unfixed errors in en.md during conversion.